sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -769,6 +769,44 @@ export interface BlobProperties {
     tagCount?: number;
 }
 
+// @public
+export interface BlobQueryCsvTextConfiguration extends BlobQueryTextConfiguration {
```

Comment:
Hi @bterlson Here is API reference change for incoming feature QuickQuery. Will create another separate API review PR for your convenience.